### DISCUSSION TITLE
优化截图录屏

### DISCUSSION THREAD

**Discussion by lizipeng0013** [linuxdeepin/developer-center#4682] [Features Request & Ideas | 特性请求 & 头脑风暴]
![录屏_VirtualBox Machine_20230612151349](https://github.com/linuxdeepin/developer-center/assets/123794418/40ce44ff-e3b1-40ae-a92d-3f5e95f22bd7)
在截图或录屏保存到本地磁盘时，同时也会自动保存剪切板，应该在设置里增加个选项，让用户自由选择是否同时保存到剪切板，现在这样一录屏或截图之前剪切板的内容就被没了，还以为出bug了

**Comment by BLumia** [comment]
cc @meiyixiang

**Comment by meiyixiang** [comment]
问题反馈给截图录屏插件项目组。

**Comment by meiyixiang** [comment]
@BLumia 我们社区自己能不能处理？否则需要等截图录屏项目组排期。

**Comment by meiyixiang** [comment]
你的意思是，不希望录屏完成后视频内容被同步到剪切板？

**Reply by lizipeng0013** [reply]
> 你的意思是，不希望录屏完成后视频内容被同步到剪切板？

给个自定义选项更好，这种已经保存到本地还同步到剪切板感觉多余了，不是每个人都需要的

**Comment by meiyixiang** [comment]
但是我实际不太理解，现在的录屏完成后在剪切面板对你的使用有什么影响？因为文件在删除后实际上剪切板中的视频也不会占用你的存储空间，只是会在剪切板中有一个占位符。

**Reply by BLumia** [reply]
按我理解是：我剪切板里可能有我之前存的东西，是可能后面会用的，结果截图后剪切板里东西变了，因而影响了使用。

**Comment by pmzgit** [comment]
![1686931149161](https://github.com/linuxdeepin/developer-center/assets/11833302/85b30bd2-1180-49d4-b745-a7d180b6a85b)

**截图撤销功能啥时候加入？**

**Comment by meiyixiang** [comment]
当前在功能页面没有提供撤销功能按钮，但快捷键ctrl+z支持类似操作，关于界面需求已经反馈給相关产品经理评估是否会在界面添加此功能按钮。
